example_content/candace.yaml

```diff
@@ -0,0 +1,6 @@
+name: Candace
+background: Candace is a computer scientist.  She graduated top of her computer science class 
+  at Stanford.  She wanted to make some money after graduation to pay off her student loans, 
+  and considered working for Meta, Google, or another big tech company, but decided that the 
+  position with Science Ventures was less evil.
+description: Candace is tall, has disheveled hair, and wears nothing but brown hoodies.
```

Comment:
aside: (and jeans) -- please be kind to us, stable diffusion image gods.

Reply:
YIKES